ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf

```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:30:00
-#PBS -l select=1:mpiprocs=11:ompthreads=1:ncpus=11:mem=80GB
+#PBS -l select=1:ncpus=11:mem=80GB
```

Comment:
Please undo and accept developer provided resources.

Reply:
undo is done